accelerator/config.c

```diff
@@ -38,7 +38,7 @@ static int get_conf_key(char const* const key) {
 }
 
 status_t cli_core_set(ta_core_t* const core, int key, char* const value) {
-  if (value == NULL && (key != CACHE && key != PROXY_API && key != QUIET && key != NO_GTTA)) {
+  if (value == NULL && (key != CACHE && key != PROXY_API && key != QUIET && key != NO_GTTA && key != RUNTIME_CLI)) {
```

Comment:
Do you think that we need to use switch case to make it more readable, since there are quite many commands without argument here